daml-assistant/get-daml.sh

```diff
@@ -4,21 +4,24 @@
 # SPDX-License-Identifier: Apache-2.0
 
 #
+# DAML is an open-source privace aware smart contract language.
 # This script downloads and installs the DAML SDK on Linux and macOS.
 # This will overwrite any existing installation in ~/.daml
+# For more information please visit daml.com and docs.daml.com
```

Comment:
https://daml.com etc - that way it's got a chance of rendering as a hyperlink if viewed on the console or a text editor.

Reply:
:+1: